xptifw/src/CMakeLists.txt

```diff
@@ -1,19 +1,11 @@
 # Fetch third party headers
 include(FetchContent)
-set(EMHASH_REPO https://github.com/ktprime/emhash)
-message(STATUS "Will fetch emhash from ${EMHASH_REPO}")
-FetchContent_Declare(emhash-headers
-  GIT_REPOSITORY    ${EMHASH_REPO}
-  GIT_TAG           4fc20c7c9b048651c03258460e4662c8d3a04520
-)
 
-FetchContent_GetProperties(emhash-headers)
-if(NOT emhash-headers_POPULATED)
-  FetchContent_Populate(emhash-headers)
+# The sycl target should download or find emhash.
+if (NOT SYCL_EMHASH_DIR)
+  message(FATAL_ERROR "emhash headers not found")
```

Comment:
This has broken UR standalone builds.

Reply:
I'll fix it, can you tell me how to reproduce it?

Reply:
Thanks, if I can't fix it within 30 min I'll revert.